packages/web-components/src/accordion/accordion/accordion-custom-icon.stories.ts

```diff
@@ -0,0 +1,135 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../../__test__/helpers.js';
+import { AccordionItemExpandIconPosition, AccordionItemSize } from '../accordion-item/accordion-item.options.js';
+import type { Accordion as FluentAccordion } from './accordion.js';
+import './define';
+import '../accordion-item/define';
```

Comment:
Are either of these actually needed? The storybook should build with a reference to all.